app/shared/utils/toast/toast.test.tsx

```diff
@@ -0,0 +1,33 @@
+import * as reactHotToast from "react-hot-toast";
+import { toast } from "./toast";
+
+jest.mock("react-hot-toast", () => ({
+  __esModule: true,
```

Comment:
precisa disso pq?👀

Reply:
Comecei usando o spyOn pra poder monitorar/mockar a função `toast` da própria lib, porém quando rodava os testes, dizia `Cannot redefine property: toast`, então pesquisando mais vi que isso se dá por conta de *re-exports* da própria lib pro jest, então uma das soluções mais difundidas foi fazer o mock da lib com esModule true (permitido es6 exports) e que queremos usar a lib de fato, então informa o `requireActual`... Acabou resolvendo o problema, mas to pesquisando mais para ver se tem outras soluções e se essa de fato é boa prática, por exemplo